modules/ROOT/pages/use-tags-in-visualizer.adoc

```diff
@@ -9,10 +9,10 @@ If a tag is assigned to an application, the application will appear in every vie
 
 == Setting the Tag Through Properties
 
-You can automate adding tags to a node by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API. 
+You can automate adding tags to a node by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API.
```

Comment:
Can a node have more than one tag? If not, then "adding tags to nodes".

Reply:
```suggestion
You can automate adding tags to nodes by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API.
```